src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const items = sourceString.split(';');
+  const result = {};
+
+  items.map((item) => {
+    const keyValue = item.split(':');
+
+    if (keyValue.length === 2) {
+      const key = keyValue[0].trim();
+      const value = keyValue[1].trim();
+
+      result[key] = value;
+    }
```

Comment:
You can simplify your code like this
```suggestion
    const [key, value] = item.split(':').map((it) => it.trim());

    if (key && value) {
      result[key] = value;
    }
```